src/edu/wright/cs/raiderplanner/controller/SettingsController.java

```diff
@@ -0,0 +1,412 @@
+import javafx.scene.input.MouseEvent;
+import javafx.scene.input.TouchEvent;
+import javafx.scene.input.TransferMode;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Background;
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.BackgroundImage;
+import javafx.scene.layout.BackgroundPosition;
+import javafx.scene.layout.BackgroundRepeat;
+import javafx.scene.layout.BackgroundSize;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.ColumnConstraints;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Pane;
+import javafx.scene.layout.Priority;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import javafx.scene.text.Font;
+import javafx.scene.text.FontPosture;
+import javafx.scene.text.FontWeight;
+import javafx.scene.text.TextAlignment;
+import javafx.stage.Screen;
+import javafx.stage.Stage;
+import javafx.util.Duration;
+import jfxtras.scene.control.agenda.Agenda;
+
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+import java.net.URL;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.Locale;
+import java.util.ResourceBundle;
+
+/**
+ * Actions associated with the settings menu and its items.
+ * Settings menu implemented in correspondence with Issue #238.
+ * 2/1/2018 - Template Created
+ * @author Clayton D. Terrill
+ */
+
+public class SettingsController implements Initializable {
+
+	/**
+	 * Initializes switch names and other buttons.
+	 * Represent states that the settings menu can be in.
+	 * May be assigned to a button in the initialize method and
+	 * what it does may be applied in the main method's switch statement.
+	 */
+	public enum Window {
+		EMPTY, ABOUT, GENERAL, ACCOUNT, THEME, NOTIFICATIONS
+	}
+
+	private Window current;
+	private boolean isNavOpen;
+	private boolean mouseDown = false;
+	private boolean initialLoad = true;
+
+	// Screen size:
+	private double screenWidth = Screen.getPrimary().getVisualBounds().getWidth();
+	private double screenHeight = Screen.getPrimary().getVisualBounds().getHeight();
+	private double screenAverage = (screenWidth + screenHeight) / 2;
+
+	// Shadows:
+	private int navShadowRadius = (int) (screenAverage * 0.03);
+	private int navShadowOffset = (int) (screenAverage * 0.01);
+	private DropShadow navShadow = new DropShadow(navShadowRadius, navShadowOffset, 0, Color.BLACK);
+	//private DropShadow notifShadow = new DropShadow(screenAverage * 0.02, 0, 0.009, Color.BLACK);
+	private DropShadow moduleDefaultShadow = new DropShadow(screenAverage * 0.005, 0, 0,
+			Color.BLACK);
+	private DropShadow moduleHoverShadow = new DropShadow(screenAverage * 0.02, 0, 0, Color.BLACK);
+	private InnerShadow modulePressedShadow = new InnerShadow(screenAverage * 0.017, 0, 0,
```

Comment:
`moduleDefaultSahdow`, `moduleHoverShadow`, and `modulePressedShadow` are never used. Should they still be here?

Reply:
Could probably be removed with no side effects. I do not believe that it would be needed to implement any settings.